packages-ts/gauntlet-terra-contracts/src/lib/multisig.ts

```diff
@@ -0,0 +1,4 @@
+export type Member = {
```

Comment:
I don't expect this type list to grow much, let's bring this type over here: https://github.com/smartcontractkit/chainlink-terra/pull/132/files#diff-efaf39312302f9a5d3ac0b8232f61b9c01fe29526694c360400282cc5f779ee2R10